app/code/EMS/Pay/Model/Method/Ideal.php

```diff
@@ -124,7 +124,7 @@ protected function _getMethodSpecificRequestFields()
     {
         $fields = [];
         $fields[Info::IDEAL_ISSUER_ID] = $this->_getIssuingBankCode();
-        $fields[Info::IDEAL_CUSTOMER_ID] = $this->_getCustomerid();
+        $fields[Info::IDEAL_CUSTOMER_ID] = $this->_getOrderId();
```

Comment:
@EvgeniyWeb4pro can you comment on this line in the code why this has been done this way - for other developers to make sense of this decision ?